topi/python/topi/arm_cpu/injective.py

```diff
@@ -62,9 +62,13 @@ def schedule_injective(outs):
     outs = [outs] if isinstance(outs, te.tensor.Tensor) else outs
     s = te.create_schedule([x.op for x in outs])
     x = outs[0]
+    ins = x.op.input_tensors
+    dtype = ins[0].dtype if len(ins) > 0 else x.dtype
+    max_vlen = 4 if dtype == 'int32' else 8
```

Comment:
Seems like 4 should be better for float32 as well. If it is, then maybe we should always use 4 instead of 8.

Reply:
I will set at 4, as a good enough value. The point is that in a network this doesn't matter, as it will get fused to the convolution op (which will decide the level of vectorization)